pages/browse-tasks.tsx

```diff
@@ -60,14 +60,16 @@ const extraColumns: GridColDef[] = [
     renderHeader: () => <TableHeader>Rating</TableHeader>,
     renderCell: (params: GridValueGetterParams) =>
       params.row.rating >= 0 ? (
-        <StyledRating
-          readOnly
-          value={params.row.rating}
-          size="large"
-          precision={0.5}
-          icon={<StarRoundedIcon fontSize="inherit" />}
-          emptyIcon={<StarOutlineRoundedIcon fontSize="inherit" />}
-        />
+        <Box ml="-7px">
```

Comment:
hm do we have to have a negative margin? honestly a nit so I don't think we need to worry about it

Reply:
Yes, without the negative margin the stars look too far to the right!